wrappers/ios_legacy/ci/build.sh

```diff
@@ -14,7 +14,7 @@ OUTPUT_DIR=/tmp/artifacts
 setup() {
     echo "ios_legacy/ci/build.sh: running setup()"
     echo "Setup rustup"
-    rustup default 1.64.0
+    rustup default 1.65.0
```

Comment:
we should try to make small PR with this change in isolation to check if this is the cause of failing ios CI - this is main suspect probably

Reply:
Looks innocent https://blog.rust-lang.org/2022/11/03/Rust-1.65.0.html#splitting-linux-debuginfo but probably main suspect here, as it's the only change dealing with linker

Reply:
I looked at the release notes. That's for linux. It was already enabled for macOS.

Reply:
This is now (finally) fixed.